src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,6 +9,7 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_BIRTHDATE = new Prefix("b/");
```

Comment:
Is birth date one word or two word? `PREFIX_BIRTH_DATE` or `Birthdate` (class name)? Personally, I don't see the current naming as a problem though, just food for thoughts.